Tests/Integration/BehaviorTests/tests/type_conversions/test/migrations/1.js

```diff
@@ -0,0 +1,9 @@
+var config = require("../config.js")
+
+var Contract = artifacts.require("./" + config.contractName + ".sol");
+var External = artifacts.require("./External.sol");
```

Comment:
How come this path is `External.sol`, not `sol_external.sol` ?

Reply:
taken from `xcall` directory. It's some odd naming scheme in truffle.